humans

```diff
@@ -4,12 +4,15 @@
 set -ue
 
 # Load configuration file
-if ! (source config.sh 2> /dev/null); then
-  echo >&2 "Could not find $(dirname "$0")/config.sh"
+cfg_file=$(dirname "$0")/config.sh
+if [ ! -f "$cfg_file" ]; then
+  echo >&2 "Could not find $cfg_file"
   echo >&2 "Run $(dirname "$0")/configure to create it"
   exit 1
 fi
 
+. "$cfg_file"
```

Comment:
Sorry to be such a stickler here, but can we use full words where possible? `config_file` reads much easier than `cfg_file` to me.